modules/local/filter_codons.nf

```diff
@@ -0,0 +1,54 @@
+process FILTER_CODONS {
+    tag "${fasta}"
+    label 'process_low'
+
+    conda (params.enable_conda ? { exit 1 "QIIME2 has no conda package" } : null)
+    container "quay.io/qiime2/core:2022.8"
+
+    input:
+    path(fasta)
+    path(asv)
+    path(dada2stats)
+
+    output:
+    path( "ASV_codon_filtered.table.tsv" )  , emit: asv
+    path( "ASV_codon_filtered.fna" ) , emit: fasta
+    path( "ASV_codon_filtered.list" ) , emit: list
+    path( "codon.filtered.stats.tsv" ), emit: stats
+    path "versions.yml"          , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    if (!params.orf_end) {
```

Comment:
Please do not use `params` in the processes but rather via `modules.config`, see https://nf-co.re/developers/modules#general. That improves readability and makes processes more transparent. Admittedly not all processes follow that, but new ones should.
alternatively use additional `input:` variable, e.g. `val(orf_end)`

Reply:
Hmm, thanks for the suggestion! I will correct that